Add a litte more compat info for WPF (#14678)
dotnet/docs: docs/desktop-wpf/migration/differences-from-net-framework.md

# Differences in .NET Core WPF

This article describes the differences between Windows Presentation Foundation (WPF) on .NET Core and .NET Framework. WPF for .NET Core is an [open-source framework](https://github.com/dotnet/wpf) forked from the original WPF for .NET Framework source code.

> There are a few features of .NET Framework that .NET Core doesn't support. Meaning, WPF for .NET Core has a few limitations you should be aware of. For more information on unsupported technologies, see [.NET Framework technologies unavailable on .NET Core](../../core/porting/net-framework-tech-unavailable.md).

Reviewer: ```suggestion
There are a few features of .NET Framework that .NET Core doesn't support. For more information on unsupported technologies, see [.NET Framework technologies unavailable on .NET Core](../../core/porting/net-framework-tech-unavailable.md).
```

## Code Access Security (CAS)

> Code Access Security (CAS) is not supported by .NET Core or WPF for .NET Core. All CAS-related functionality is treated under the assumption of full-trust. WPF for .NET Core has been removing CAS-related code. The public API surface of these types still exists to ensure that calls into these types succeed. However, these calls don't actually do anything.

Reviewer: ```suggestion
Code Access Security (CAS) is not supported by .NET Core or WPF for .NET Core. All CAS-related functionality is treated under the assumption of full-trust. WPF for .NET Core removes CAS-related code. The public API surface of these types still exists to ensure that calls into these types succeed. However, these calls don't actually do anything.
```

> - `PrivateAccessToTypeName`
> - `AssemblyNameString`

## Next steps

- [Learn how to port a .NET Framework WPF app to .NET Core.][migrate-project]

> [migrate-project]: convert-project-from-net-framework.md

Author: ```suggestion

```
